prisma/schema.prisma

```diff
@@ -14,7 +14,7 @@ datasource db {
 }
 
 model User{
-  id Int @id @default(autoincrement())
+  id String @id 
```

Comment:
What was the reason to replace the userID with a string type? Do we expect alphabetical characters in the userID?

Reply:
Yes, I wanted the table to match firebase ID's which have characters in them.